src/components/VInput/VInput.vue

```diff
@@ -69,7 +69,8 @@
       <select
         v-if="'select' === $attrs.type"
         ref="input"
-        v-model="localValue"
+        :value="localValue"
+        @input.prevent="localValue = $event.target.value"
```

Comment:
This looks fine, but why did you add the `prevent` modifier?

Reply:
Not a real reason to be honest

Reply:
Ok. Wasn't sure if this was related to the bug. I'll merge this in, but I'll have to manually create the next release. I still don't have the release system automated.